katportalclient/client.py

```diff
@@ -1643,17 +1643,18 @@ def sensor_subarray_lookup(self, sub_nr, component, sensor,
 
         katcp_name: bool
             True to return the katcp name, False to return the fully qualified
-            python sensor name. Default is False.
+            Python sensor name. Default is False.
 
```

Comment:
The `sub_nr` parameter has been moved to the end of the method definition, so please move it in the docstring as well.
Also update the description to say the default is `None`, in which case it will use the subarray number based on the connection URL.